packages/core/src/FrameLoop.ts

```diff
@@ -189,9 +189,7 @@ export class FrameLoop {
       else {
         velocity = animated.lastVelocity !== void 0 ? animated.lastVelocity : v0
 
-        const w0 = Math.sqrt(config.tension! / config.mass!)
-
-        const dt = 100 / w0
+        const dt = 0.05 / config.w0
```

Comment:
This changed from `100 / w0` to `0.05 / (w0 / 1000)`. Was not using `0.1` intended?

Reply:
I think I will stabilize this later, I need to run some more tests. Precision with `0.05` or `0.1` isn't so great compared to analytical.

<img width="826" alt="springperf" src="https://user-images.githubusercontent.com/5003380/64127964-4233a900-cdb4-11e9-93a6-f73530b3e480.png">

As you can see from above, the yellow curves (large steps) move (way) too fast compared to analytical / 1ms Euler  (performance gains seem to be important though on the first run).

Reply:
@aleclarson @drcmda pretty accurate bench I think, results vary depending on the spring config though.

https://jsperf.com/react-spring/16

Reply:
Nice. Would be great if we had a bar graph that compared different spring configs, but idk how easy that would be to put together.

Reply:
@aleclarson ok so here is a basic performance test comparing Euler to Analytical (longer bars are better): https://codesandbox.io/s/goofy-blackwell-66tpc

Results for `step = 0.05/w0`
<img width="809" alt="step1" src="https://user-images.githubusercontent.com/5003380/64186716-99e31a80-ce6f-11e9-9e5f-7cb3095c6ab9.png">

Results for `step = 0.025/w0`
<img width="804" alt="step2" src="https://user-images.githubusercontent.com/5003380/64186739-a36c8280-ce6f-11e9-9db7-5b49aa47620d.png">

**🚨&nbsp;This is not _per cycle_ performance: one algorithm might be faster and more accurate, which might result in the spring to come to rest later, and therefore the test to appear slower&nbsp;🚨**

So depending on the step we end up choosing, there will be a significant difference with performance compared to the analytical algorithm. Two things to note:

- The longer the step, the lesser the precision (see previous tool) 
- I'm not sure how interruptible the analytical algorithm is

Reply:
Interestingly, running this on my iPhone gives significantly different results.

![image](https://user-images.githubusercontent.com/5003380/64205965-8482e600-ce99-11e9-9fd5-8b2c106f6cf5.png)

Reply:
I've also integrated `RK4` algorithm and an option to limit the benchmark to the first 30 cycles. That will actually measure the raw performance of the algorithm per frame.

<img width="1040" alt="image" src="https://user-images.githubusercontent.com/5003380/64248525-9f913c80-cf11-11e9-83e9-c0746d006d48.png">

My first conclusion is that the analytical algorithm is really impressive.